src/shared/lib/resources/cwu-opportunity.ts

```diff
@@ -0,0 +1,41 @@
+import { Addendum } from 'shared/lib/resources/addendum';
+import { FileRecord } from 'shared/lib/resources/file';
+import { User } from 'shared/lib/resources/user';
+import { Id } from 'shared/lib/types';
+
+export enum OpportunityStatus {
+  Draft = 'DRAFT',
+  Published = 'PUBLISHED',
+  Evaluation = 'EVALUATION',
+  Awarded = 'AWARDED',
+  Suspended = 'SUSPENDED',
+  Canceled = 'CANCELED'
+}
+
+export interface CWUOpportunity {
+  id: Id;
+  createdAt: Date;
+  createdBy: User;
+  updatedAt: Date;
+  updatedBy: User;
```

Comment:
`createdBy` and `updatedBy` should be optional — `undefined` for non-admin users and for users that didn't create/update the opportunity.